app/dao/notifications_dao.py

```diff
@@ -217,23 +216,29 @@ def get_notification_billable_unit_count_per_month(service_id, year):
     start, end = get_financial_year(year)
 
     notifications = db.session.query(
-        NotificationHistory.created_at,
-        NotificationHistory.billable_units
-    ).order_by(
-        NotificationHistory.created_at
+        func.date_trunc("month",
+                        (NotificationHistory.created_at +
+                         extract("timezone", func.timezone("Europe/London",
+                                                           NotificationHistory.created_at)) * timedelta(seconds=1))),
```

Comment:
if you make this a variable above then we won't need the ridiculous group_by and order_by duplication.

also this really needs its own comment explaining what it's doing i think, given the amount of time and trouble we had figuring out the SQL in the first place